.github/workflows/test-environment.yml

```diff
@@ -0,0 +1,102 @@
+name: sanity workflow
+
+on:
+  # Ability to execute on demand
+  workflow_dispatch:
```

Comment:
Are we going to change it later on?

Reply:
We are going to add also scheduled option when the workflow will be executed automatically.